### DISCUSSION TITLE
Pass Previous Frame In Video

### DISCUSSION THREAD

**Discussion by RFedz007989** [light-and-ray/sd-webui-replacer#53] [General]
Hi,

Can anyone pls explain what is pass previous frame in Video & how to do it?

Thank You

**Comment by light-and-ray** [comment]
Pass the Previous Frame Into ControlNet - after a frame was generated, it will be used in ControlNet input to control the next frame generation. Use IP-Adapter or similar. In can increase consistency

![result4](https://github.com/light-and-ray/sd-webui-replacer/assets/33491867/2aefe1e1-4838-4763-a8ac-fdc073d84032)

**Comment by RFedz007989** [comment]
> Pass the Previous Frame Into ControlNet - after a frame was generated, it will be used in ControlNet input to control the next frame generation. Use IP-Adapter or similar. In can increase consistency
> 
> ![result4](https://private-user-images.githubusercontent.com/33491867/318015486-2aefe1e1-4838-4763-a8ac-fdc073d84032.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3MDg3MjAsIm5iZiI6MTcxMTcwODQyMCwicGF0aCI6Ii8zMzQ5MTg2Ny8zMTgwMTU0ODYtMmFlZmUxZTEtNDgzOC00NzYzLWE4YWMtZmRjMDczZDg0MDMyLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI5VDEwMzM0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM5MTNlMDlhMTA2ZWQwN2IzY2I3YmFlOTA1MmRiODQxNzg2YWIxNzRkMzA1ZTczZWZmMzI2OTVlZTQwMzViZTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.22bPA4ABWoC46V34XuUTUUg1wr2eQXHtVBy8q8oSUsY)

Thanks a lot for your quick & detail explanation.
